### ISSUE TITLE
Default URL doesn't redirect to category page

### ISSUE THREAD

**Issue by reeversedev** [reeversedev/devdevdev#22]
The index URL of the project doesn't take the user to the category page. 
The URL should ideally be: https://devdevdev.now.sh/category/new-in

Hence, the following behavior is there.
![Screen Shot 2021-01-03 at 1 22 54 PM](https://user-images.githubusercontent.com/21155917/103474157-07d28580-4dc7-11eb-9ab1-6641496868f9.png)

**Comment by gkanishk** [comment]
I can take this up, I am planning to redirect that on page load.
```
  useEffect(() => {
    const {pathname} = Router
    if(pathname == '/' ){
        Router.push('/category/new-in')
    }
  });
 ```
 If this looks good then I can raise a PR

**Comment by reeversedev** [comment]
@gkanishk  Yes, I think that should solve it. Create PR.

**Comment by gkanishk** [comment]
Great @reeversedev !! I have raised the PR #24 kindly review and let me know if there's any change required.

**Comment by reeversedev** [comment]
Fixed in #24
